src/components/tab/tab.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from 'react';
+import { SizeType } from '../../core/types/size';
+import IonIcon from '../icons/icons';
+import { iconType } from '../icons/svgs/icons';
+import { TabStyles } from './styles';
+
+type StitchesTabProps = React.ComponentProps<typeof TabStyles>;
+
+type TabSizes = Exclude<SizeType, 'xs'>;
```

Comment:
I believe that is better use Omit.

https://timmousk.com/blog/typescript-omit/#:~:text=As%20you%20can%20see%2C%20omitting,literal%2C%20use%20the%20Exclude%20type.